tests/data/scad/misc/utf8-import.scad

```diff
@@ -0,0 +1,17 @@
+// Test various imports of files with non-ASCII names.
+// Note that the resulting image is actually of no interest.
+// However, any interesting errors in these imports happen
+// during geometry processing, so we can't just be an
+// echo, csg, or ast test - we have to actually do the
+// geometry processing.
+import("../../dxf/☠-2D.dxf");
+import("../../stl/☠-3D.stl");
+import("../../off/☠-3D.off");
+import("../../obj/☠-3D.obj");
+
+// These fail because Windows fopen() can't handle UTF-8 paths.

```

Comment:
..but it only fails in MXE mode?

Reply:
Since we don't test MXE (yet), I'll go ahead and add those.

Reply:
Note that for this test the actual resulting image isn't all that interesting; what's interesting is whether we get any error messages.  (Hmm.  I need to check whether the test actually detects error messages.)